core/trino-main/src/main/java/io/trino/sql/planner/plan/TableWriterNode.java

```diff
@@ -471,6 +522,12 @@ public String toString()
         {
             return handle.map(Object::toString).orElse("[]");
         }
+
+        @Override
+        public boolean supportsReportingWrittenBytes(Metadata metadata, Session session)
+        {
+            return false;
```

Comment:
Why does it return `false`? Should it throw `UnsupportedOperationException` instead? If this method is still being called at any point does it make sense to return a meaningful value?